verifier/src/lib.rs

```diff
@@ -169,19 +236,27 @@ impl<T, L: Verifier<T>, R: Verifier<T>> Verifier<T> for And<L, R> {
 }
 
 /// An error that occurs during an `or` operation.
-#[derive(Debug)]
+#[derive(Debug, Clone)]
 pub struct OrError<L, R> {
-    _left: CtOption<L>,
-    _right: CtOption<R>,
+    left: CtOption<L>,
+    right: CtOption<R>,
 }
 
 impl<L, R> OrError<L, R> {
     /// Create a new instance
     pub fn new(left: CtOption<L>, right: CtOption<R>) -> Self {
-        Self {
-            _left: left,
-            _right: right,
-        }
+        Self { left, right }
+    }
+}
+
+impl<L: DisplayableError, R: DisplayableError> DisplayableError for OrError<L, R> {}
+
+impl<L: DisplayableError, R: DisplayableError> Display for OrError<L, R> {
+    fn fmt(&self, f: &mut Formatter) -> core::fmt::Result {
+        f.debug_struct("OrError")
+            .field("left", &format_args!("{:#}", self.left.display()))
+            .field("right", &format_args!("{:#}", self.right.display()))
+            .finish()
```

Comment:
Please don't use `debug_struct` for `Display`. Recommend:

```
Neither of these conditions were satisfied:
  - {left:#}
  - {right:#}
```

Reply:
updated but don't think it's quite what was desired